### DISCUSSION TITLE
20230817 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by cgbosse** [nautilus-cyberneering/.github#54] [Meetings]
Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- CGarnham(@da2ce7)  - 
- JCelano(@josecelano) -  
- CBosse(@cgbosse) - x
- MMelchor(@mario-nt) - x
- GByrne(@grmbyrn) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by mario-nt** [comment]
### Paperwork/Other
- Reviewing some mandatory documents and courses
- Setting up a new app for internal communication
### Torrust
- Quick overview of the app (user creation, torrent upload and creation, etc)
- Finished setting up the development environment (IDE, dependencies, etc)
- Quick review of Jose's [E2E testing commit with cypress](https://github.com/torrust/torrust-index-frontend/pull/222)
- Worked on[ issue #205](https://github.com/torrust/torrust-index-frontend/issues/205) The downloaded torrent filename is the torrent title instead of the original torrent file name
- Torrent 101 (researching what a tracker is, what the bittorrent protocol is, how to torrent in a secure way, etc)
### Training
- Reading the [Rust programming book](https://doc.rust-lang.org/book/)

**Comment by grmbyrn** [comment]
## Admin
* Health and safety course
* Set up app for internal communication
* Review documents from health and safety course instructor

## Svelte
* Research useful resources for learning Svelte, such as tutorials, project-based learning and testing
* Complete Svelte tutorial in official Svelte docs https://learn.svelte.dev/tutorial/welcome-to-svelte
* Complete Feedback App in Traversy Media video: https://www.youtube.com/watch?v=3TVy6GdtNuQ&ab_channel=TraversyMedia (GitHub repo: https://github.com/grmbyrn/feedback-app-svelte)
* Complete calculator and image gallery projects in https://www.youtube.com/watch?v=o6VwbXSjM08&t=6096s&ab_channel=LaithAcademy (GitHub: https://github.com/grmbyrn/calculator-svelte)

## Diary
* Began writing a diary to keep track of progress on Medium: https://medium.com/@grmbrn89
